client/state/account-recovery/reset/test/reducer.js

```diff
@@ -24,6 +35,29 @@ describe( '#account-recovery/reset reducer', () => {
 		assert.isTrue( state.options.isRequesting );
 	} );
 
+	const hasItemsState = {
+		options: {
+			items: fetchedOptions,
+		},
+	};
+
+	it( 'ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST action should wipe the previous items.', () => {
```

Comment:
Suggested change "ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST action should ~wipe~ `delete` the previous items."